src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -137,16 +137,29 @@ public async Task<ActionResult<List<DataFileInfo>>> GetDataFileInfo(Guid release
         [ProducesResponseType(404)]
         [RequestSizeLimit(int.MaxValue)]
         [RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = int.MaxValue)]
-        public async Task<ActionResult<DataFileInfo>> AddDataZipFileAsync(Guid releaseVersionId,
+        public async Task<ActionResult<DataFileInfo>> UploadAsZip(Guid releaseVersionId,
             [FromQuery(Name = "replacingFileId")] Guid? replacingFileId,
-            [FromQuery(Name = "title")] string subjectName,
+            [FromQuery(Name = "title")] string dataSetTitle,
             IFormFile zipFile)
         {
             return await _releaseDataFileService
                 .UploadAsZip(releaseVersionId: releaseVersionId,
                     zipFormFile: zipFile,
-                    replacingFileId: replacingFileId,
-                    subjectName: subjectName)
+                    dataSetTitle: dataSetTitle,
+                    replacingFileId: replacingFileId)
+                .HandleFailuresOrOk();
+        }
+
+        [HttpPost("release/{releaseVersionId:guid}/bulk-zip-data")]
+        [Produces("application/json")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        [RequestSizeLimit(int.MaxValue)]
```

Comment:
> Is the intention to limit the request size to 2Gb here?

Frankly, I was just copying the other endpoints. I could switch to `DisableRequestSizeLimit` as you suggest, but it the current attribute these has been in place for a long time and I'm wary of changing something that could potentially break something, without spending the appropriate time investigating it.

My inclination is to create a ticket to do this separately. But if you're confident in the change, I'm happy to do it.

> Also just wondering if you have done any testing of large zip file uploads for this new bulk upload feature?

I haven't - I'm inclined to leave that to the testers. If this goes to the dev branch and we do have a problem, a small migration will be required to revert the change, but otherwise it should be fine.

Reply:
I've switched to using `[DisableRequestSizeLimit]`, and made the same change across the whole codebase.

I'll leave a note to test for large files on the ticket (although I'm sure they would anyway).